app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,2 @@
+class CommentsController < ActionController
+end
```

Comment:
- [ ] Kindly ensure to remove the CommentsController, like indicated in the previous review.

Reply:
well resolved! ✔️